e2e/pages/jsonld.tsx

```diff
@@ -195,6 +195,12 @@ export default () => (
           },
         },
       ]}
+      aggregateOffer={{
+        priceCurrency: 'USD',
+        lowPrice: '119.99',
+        highPrice: '199.99',
```

Comment:
The `highPrice` is `139.99`

Reply:
I used the data from google developers docs.
FIXED ✔️